src/main/java/seedu/duke/shooter/Shooter.java

```diff
@@ -0,0 +1,37 @@
+package seedu.duke.shooter;
+
+public class Shooter {
+    public String name;
+    private int power;
+    private int skill;
+
+    public Shooter(String name) {
+        this.name = name;
+    }
+    public void printSelfInfo(){
+        System.out.println("--------Player Info--------");
+        System.out.println("Name: "+this.name);
+        printPower();
+        printSkill();
+        System.out.println("---------------------------");
+    }
+
+    protected void printPower(){
+
+    }
```

Comment:
Why are the implementations for some of the functions empty?

Reply:
https://nus-cs2113-ay2324s2.github.io/website/schedule/week4/topics.html#W4-1

Reply:
You should indicate in the comments that these functions are being overridden by the children class, otherwise it can be confusing because it is unclear why these methods are empty while the others in the same class are not.